app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= bootstrap_form_with(model: category, html: { class: 'p-2' }) do |form| %>
+  <% if category.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(category.errors.count, "error") %> prohibited this category from being saved:</h2>
+
+      <ul>
+        <% category.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.select :icon,  [ ['👔'], ['👗'], ['🏠'], ['❤️'], ['✈️'], ['🐶'], ['👨‍👩‍👧‍👦'], ['🎉'], ['🛒'], ['🍕'], ['🗺️'], ['🏥'], ['🏈'], ['⛺'], ['⛱️'], ['🚲'], ['🔨'], ['📆'], ['🔒'], ['📌'], ['🕙'], ['🧹']] %>
+  </div>
+
+  <div>
+    <%= form.submit %>
+  </div>
+<% end %>
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20user%20clicks%20(or%20taps)%20the%20%22Save%22%20button%20to%20create%20the%20new%20category%2C%20and%20is%20taken%20to%20the%20home%20page%20on%20success.), after creating a new category, and the category is saved, the user should be re-directed to the categories page. Instead the user is re-directed to the transactions page for that new category. Kindly fix this issue.